Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3.10-slim-bookworm
 
 RUN apt-get clean
 RUN apt-get update
-RUN apt-get install -y curl pkg-config build-essential libnss-myhostname
+RUN apt-get install -y curl pkg-config build-essential libnss-myhostname automake libtool libffi-dev libgmp-dev
```

Comment:
Out of scope for this PR I guess, but this is not the most effective way to use apt in a Docker build. Using it like this saves some space in the image: https://github.com/BoltzExchange/boltz-backend/blob/master/docker/boltz/Dockerfile#L30

Also why are dependencies needed to *build* secp256k1 included in the image?

Reply:
https://github.com/lnbits/lnbits/actions/runs/9855301099

actions fails building it :(